### DISCUSSION TITLE
amdgpu kernel 6.10 complete system freezes with vulkan.

### DISCUSSION THREAD

**Discussion by voidastro4** [mpv-player/mpv#14659] [General]
Ryzen 5000 U-series, amdgpu

I don't know where to report this so I will just leave it here.

Starting on kernel 6.10 when fullscreening mpv with hwdec=vulkan active there is a chance that there will be a complete system freeze for up to a minute that leaves no trace in dmesg (frozen screens and unresponsive keyboard). Audio is active during the freeze though there are audible artifacts.

**Comment by llyyr** [comment]
Not our bug. There's about half a dozen different amdgpu bugs that currently cause a Kernel panic inside an IRQ handler.

You can see if this patch helps you https://gitlab.freedesktop.org/drm/amd/-/issues/3142#note_2428259 or just downgrade to 6.6 LTS branch

**Comment by voidastro4** [comment]
Probably different, I had no issues on 6.9 and that's what I downgraded to.

**Comment by brauliobo** [comment]
I'm still having freezes on all vulkan applications (chrome if enabled, nautilus, etc) with KDE on X11 and Wayland, kernel 6.11.8, Ryzen 9700x integrated GPU

**Reply by llyyr** [reply]
Why do you think mpv is the place to report this? https://gitlab.freedesktop.org/drm/amd/-/issues/

**Reply by voidastro4** [reply]
> Why do you think mpv is the place to report this? https://gitlab.freedesktop.org/drm/amd/-/issues/

to be fair in my experience they dont do anything.
Issue is present in 6.11 on my end too

https://gitlab.freedesktop.org/drm/amd/-/issues/3656
